Documentation/GeneralConventions/ReadmeFile.rst

```diff
@@ -0,0 +1,95 @@
+
+.. index:: File structure; README.rst, README.rst
+.. _readme-rst:
+.. _about-file:
+
+=======================================
+:file:`README.rst` or :file:`README.md`
+=======================================
+
+Full documentation contains both a README.rst and a Documentation/Index.rst
```

Comment:
```suggestion
Full documentation contains both a :file:`README.rst` and a :file:`Documentation/Index.rst`
```

I already mentioned, I think the phrase "full documentation" may be misleading, as a full documentation can also be achieved with a README.md. But we can change this later.